common/sgx/tlsverifier.c

```diff
@@ -172,3 +202,136 @@ oe_result_t oe_verify_attestation_certificate(
     oe_free(report);
     return result;
 }
+
+/**
+ * oe_verify_attestation_certificate_with_evidence
+ *
+ * This function performs a custom validation on the input certificate. This
+ * validation includes extracting attestation evidence extension from the
+ * certificate before validating this evidence. An optional
+ * claim_verify_callback could be passed in for a calling client to further
+ * validate the claims of the enclave creating the certificate.
+ * @param[in] cert_in_der a pointer to buffer holding certificate contents
```

Comment:
Recommend spelling out "cert", "pub", "buf" throughout this file per style guidelines

Reply:
Will keep to be compatible with existing code base.